<launch>
  <!-- <node pkg="robot_localization" type="ekf_localization_node" name="ekf_mycar" clear_params="true"> -->
  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_mycar" >
    <rosparam command="load" file="$(find sdc)/config/ekf.yaml" />

     <!-- Placeholder for output topic remapping -->
    <!-- <remap from="odometry/filtered" to="odom/filtered"/> -->
    <!-- <remap from="accel/filtered" to=""/> -->
    <remap from="odometry/filtered" to="amcl_ekf"/>
    <!-- <remap from="freq_pose" to="amcl_ekf"/> -->
  </node>
</launch>